vllm/core/block/prefix_caching_block.py

```diff
@@ -293,10 +296,28 @@ def clear_copy_on_writes(self) -> Dict[BlockId, List[BlockId]]:
         """
         return self._cow_tracker.clear_cows()
 
-    def mark_blocks_as_computed(self) -> None:
-        """Mark blocks as computed, used in prefix caching."""
-        # TODO Track computed blocks.
-        pass
+    def mark_blocks_as_accessed(self, block_ids: List[int],
+                                now: float) -> None:
+        """Mark blocks as accessed, used in prefix caching."""
+
+        for block_id in block_ids:
+            if block_id in self._blocks:
+                self._blocks[block_id].last_accessed = now
+
+    def mark_blocks_as_computed(self, seq_block_ids: List[List[int]]) -> None:
+        """Mark immutable blocks as computed, used in prefix caching.
+
+        Only mark those immutable block as computed.
+        """
+        for block_ids in seq_block_ids:
+            block_cnt = 0
```

Comment:
why do we need to track `block_cnt`?

Reply:
This is no needed and forget to delete, which served for num_token computation